readme.md

```diff
@@ -26,15 +26,17 @@ Unlike cookies and local storage, fingerprint stays the same in incognito/privat
 ```html
 <script>
   function initFingerprintJS() {
-    FingerprintJS.load().then(fp => {
-      // The FingerprintJS agent is ready.
-      // Get a visitor identifier when you'd like to.
-      fp.get().then(result => {
+    // We recommend to call `load` at application startup.
+    const fpPromise = FingerprintJS.load()
+
+    // Get a visitor identifier when you need it.
+    fpPromise
+      .then(fp => fp.get)
+      .then(result => {
         // This is the visitor identifier:
         const visitorId = result.visitorId;
         console.log(visitorId);
       });
-    });
   }
```

Comment:
Having a big timespan between starting agent and getting an identifier increases the chance of getting the real screen frame size (the frame between screen resolution and available screen resolution).